modules/utxo-lib/src/bitgo/UtxoPsbt.ts

```diff
@@ -241,6 +253,24 @@ export class UtxoPsbt<Tx extends UtxoTransaction<bigint>> extends Psbt {
     return results.every((res) => res === true);
```

Comment:
What's interesting is that `results.every()` returns `true` for empty arrays, which IMO is a bug even without this change. Maybe we should change this to `0 < results.length && results.every`

Reply:
actually to stay compatible with with https://github.com/BitGo/bitcoinjs-lib/blob/master/ts_src/psbt.ts#L435 let's leave it as-is